src/models/disassemblyoutput.cpp

```diff
@@ -238,19 +261,23 @@ DisassemblyOutput::ObjectdumpOutput DisassemblyOutput::objdumpParse(const QByteA
         //    |   64 a3 ....
         //    \-> 65 23 ....
         // so we can simply skip all characters until we meet a letter or a number
-        const auto branchVisualisation = [&]() -> QStringView {
-            if (!addr)
-                return {};
-            auto firstHexIt = std::find_if(lineRest.cbegin(), lineRest.cend(), isHexCharacter);
-            auto size = std::distance(lineRest.cbegin(), firstHexIt);
-            auto ret = lineRest.mid(0, size);
-            lineRest = lineRest.mid(size);
-            return ret;
+        struct BranchesAndHexdump
+        {
+            QString branchVisualisation;
+            QString hexdump;
+        };
+        const auto [branchVisualisation, hexdump] = [branchesAndHex = parts.value(1)]() -> BranchesAndHexdump {
+            auto firstHexIt = std::find_if(branchesAndHex.cbegin(), branchesAndHex.cend(), isHexCharacter);
+            auto size = std::distance(branchesAndHex.cbegin(), firstHexIt);
+            auto branchVisualisation = branchesAndHex.mid(0, size);
+            auto hexdump = branchesAndHex.mid(size);
+            return {branchVisualisation, hexdump};
```

Comment:
Can you use `return {branchVisualisation, hexdump..trimmed()};` here? Otherwise we have trailing spaces, leading to eliding symbol "..." (and unnecessary amount of trailing spaces kept in memory).